### ISSUE TITLE
Add support for Null Safety

### ISSUE THREAD

**Issue by RegisSaffi** [Vanethos/flutter_notification_permissions#37]
Add Support for null safety to be able to upgrade to Flutter 2

**Comment by Aulig** [comment]
Null safety is already supported on master, @Vanethos just needs to publish it to pub.dev still
Use
```
  notification_permissions:
    git:
      url: https://github.com/Vanethos/flutter_notification_permissions
```
for now

**Comment by RegisSaffi** [comment]
Okay, good to know.

So, when can we expect it to be published on pub? We will appreciate that!


Regis Imuduhe Safari

**Comment by Vanethos** [comment]
Yes!

I'm deeply sorry for taking so long, but unfortunately I wasn't able to do it before

Published v0.5.0 🎉
